internal/deploy/docker.go

```diff
@@ -175,8 +167,9 @@ func (c *dockerDeploymentManifest) Logs(service ServiceRequest) error {
 }
 
 // Remove remove services from deployment
-func (c *dockerDeploymentManifest) Remove(services []ServiceRequest, env map[string]string) error {
-	for _, service := range services[1:] {
+func (c *dockerDeploymentManifest) Remove(profile ServiceRequest, services []ServiceRequest, env map[string]string) error {
+	// TODO: profile is not used because we are using the docker client, not docker-compose, to reach the service
```

Comment:
Do you want to retry with docker-compose? This was the section that was being flaky for me during removing of services

Reply:
This method is used only once: https://github.com/elastic/e2e-testing/blob/master/e2e/_suites/fleet/fleet.go#L98. I think we were still using it before these changes.